src/roles/common/tasks/netconf-process-vars.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Find Netconf RPM location
+  include_role:
+    name: common
+    tasks_from: find-file.yml
+  vars:
+    dir_name: "{{ nuage_unzipped_files_dir }}"
+    pattern: "nuage-netconf-*.rpm"
+
+- name: Register Netconf RPM variables with proper path and image locations for use in other locations
+  set_fact:
+    image_path: "{{ file_path }}"
+    image_file_name: "{{ file_name }}"
+
+- name: Create host_vars files for Netconf server
```

Comment:
Why aren't we using write-host-files like the other components? We've never added a prefix to the host_vars file names before. It seems to me that this practice would screw up our ability to reference host_vars properties based on inventory_hostname.

Reply:
We had to do a special case for this for one reason. The VM name they specify could be same as one of the other VMs for the NUH or VNSUtil and we will override the host vars property of that VM. So this has to be done.